meinberlin/apps/livequestions/api.py

```diff
@@ -36,3 +39,9 @@ def get_queryset(self):
         ):
             live_questions = live_questions.filter(is_hidden=False)
         return live_questions
+
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
I am bit unhappy with this, because I have to get the category manually, but didnt know how else to solve it..